modules/zrtp/zrtp.c

```diff
@@ -35,10 +42,14 @@ enum {
 
 struct menc_sess {
 	zrtp_session_t *zrtp_session;
+	menc_error_h *errorh;
+	void *arg;
+	struct tmr abort_timer;
```

Comment:
you should call `tmr_cancel` on this new timer in the destructor, even if it is not fired.